venues/ICLR.cc/2019/Workshop/DeepGenStruct/python/config.py

```diff
@@ -18,8 +18,11 @@ def get_conference(client):
     'website': 'https://deep-gen-struct.github.io/index.html',
     'location': 'New Orleans, Louisiana, United States'
     })
+    builder.set_authorpage_header({
+        'title': 'Author Console',
+        'instructions': 'This are the instructions',
+        'schedule': 'To be Defined'
+    })
```

Comment:
Instructions should be blank?  
schedule should capitalize 'be' or lower 'defined'

Reply:
Fixed